rest/taskrouter/activities/list/get/example-1/example-1.3.x.js

```diff
@@ -0,0 +1,14 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These consts are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+const workspaceSid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const client = require('twilio')(accountSid, authToken);
+
+client.taskrouter.v1
+  .workspaces(workspaceSid)
+  .activities
+  .list()
```

Comment:
what do you think about using the #each method provided in the sdk? https://github.com/TwilioDevEd/api-snippets/blob/next-gen-node-snippets-rest-part-1/rest/addresses/list-get-example-2/list-get-example-2.3.x.js#L9